server/constants/paths.ts

```diff
@@ -0,0 +1,16 @@
+const paths = {
```

Comment:
Most of the rest of the changes in this PR are just updating everywhere else to use these constant values instead of hard-coded strings.